src/routes/transactions/entities/queued-items/transaction-queued-item.entity.ts

```diff
@@ -0,0 +1,17 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { ConflictType } from '../conflict-type.entity';
+import { QueuedItem } from '../queued-item.entity';
+import { Transaction } from '../transaction.entity';
+
+export class TransactionQueuedItem extends QueuedItem {
+  @ApiProperty()
+  transaction: Transaction;
+  @ApiProperty()
+  conflictType: string;
+
+  constructor(transaction: Transaction, conflictType: ConflictType) {
```

Comment:
`module-transaction.entity`  and `multisig-transaction.entity` and this one are equals, maybe we can refactor it in one entity.

Reply:
Yes, in fact, I had that in a TODO note to propose it in a separate PR 😄. 

In the Rust implementation, I think this is named simply `Transaction`, and Nest's `Transaction` is a `TransactionSummary`in Rust impl. What do you think about following that names? Separate PR or do you prefer to do it here? @moisses89 @fmrsabino

Reply:
Separate PR and follow the Rust names are ok for me.